src/components/PhotoDetails.vue

```diff
@@ -80,6 +80,28 @@
              Add to list
           </a>
         </section>
+        <section class="photo_usage">
```

Comment:
I think you could reuse the `SocialShareButtons` component defined [here](https://github.com/creativecommons/cccatalog-frontend/blob/master/src/components/SocialShareButtons.vue)